### PR TITLE
Hello Microverse Project Pull Request

### PR DESCRIPTION
**Pull request by Ewoskyx**
Changes are done on the following file(s) as suggested by the reviewer.

-  README.md (Getting started and Live demo explanations are updated)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,41 @@
...
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML5 & CSS3
 
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
```
Comment (SajjadAhmad14) [inline README.md:19]: Please remove the Live demo as it does not have an active link.

File: README.md [hunk 2]
```diff
@@ -1,75 +1,41 @@
...
-
-
-## Getting Started
```
Comment (SajjadAhmad14) [inline README.md:22]: Please add instructions to run the project locally for non-technical people.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (SajjadAhmad14) [inline index.html:1]: PR should have a descriptive title and summary.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Evren Ersoy, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Ewoskyx,** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._
- Like the previous TSE mentioned that the **PR** should be descriptive.

mmmmm
pppp


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZeenatLawal — APPROVED** [review]
Hi @Ewoskyx, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**